www/bubm02/cv01/index.php

```diff
@@ -0,0 +1,67 @@
+<?php
+$avatar = "logo.png";
+$name = "Mykhailo";
+$surname = "Bubnov";
+$age = 19;
+$position = "Software Developer";
+$firmName = "Sefira spol.";
+$street = "A. Staška";
+$streetNumber1 = '2027';
+$streetNumber2 = '/77';
+$city = "Praha 4";
+$phone = "+420 777 777 777";
+$email = "applemix00@gmail.com";
+$web = "https://www.sefira.cz/";
+$lookingForJob = false;
+if ($lookingForJob) {
+    $lookingForJob = "Available for contracts";
+} else {
+    $lookingForJob = "Not available for contracts";
+}
```

Comment:
Why not ternary operator?  
`$available ? 'Available for contracts' : 'Not available for contracts'` do html kódu